test/service/actions/channels.test.js

```diff
@@ -25,6 +25,18 @@ describe('Actions.Channels', () => {
         await TestHelper.basicClient.logout();
     });
 
+    it('selectChannel', async () => {
+        await TestHelper.initBasic(Client);
+        const store = configureStore();
```

Comment:
The above two lines aren't necessary as the TestHelper and the stores are initialized in the `before` and `beforeEach` hooks